Source/Blazorise/Components/Table/TableHeader.razor.cs

```diff
@@ -22,6 +22,7 @@ public partial class TableHeader : BaseDraggableComponent
         protected override void BuildClasses( ClassBuilder builder )
         {
             builder.Append( ClassProvider.TableHeader() );
+            builder.Append( "b-table-header-theme", ThemeContrast != ThemeContrast.None );
```

Comment:
I would like to move `b-table-header-theme` under the ClassProvider. Then name it according to the current provider naming convention. Eg. `table-header-theme` for Bootstrap, `ant-table-header-theme` for AntDesign, etc.

Reply:
Damn, and here I was trying to be lazy!! Damn us programmers :D Will do. :)

Reply:
Should we just re-use TableHeaderThemeContrast then?

Reply:
In what way to reuse it?

Reply:
![image](https://user-images.githubusercontent.com/22283161/175828631-cfc4878a-b4d7-4dd6-a6ff-8176b1663f81.png)